test/phpunit/CodingPhpTest.php

```diff
@@ -188,10 +187,10 @@ public function testPHP()
 						break;
 					}
 					//print __METHOD__." Result for checking we don't have non escaped string in sql requests for file ".$file."\n";
-					$this->assertTrue($ok, 'Found string GETPOST into a .class.php file in '.$file['relativename'].'.');
+					//					$this->assertTrue($ok, 'Found string GETPOST in a .class.php file in '.$file['relativename'].'.');
```

Comment:
Why disabling this assert  ?

Reply:
I probably disabled it during debug and forgot to remove the comment.  Restored it.